src/app/components/search-bar/search-bar.component.scss

```
@@ -91,4 +123,170 @@
+
+  .custom-control--nested {
+    margin-top: -0.25rem;
+  }
+
+  .custom-autocomplete__input-wrap {
+    position: relative;
+  }
+
+  .custom-autocomplete__input {
+    @apply w-full rounded-lg border border-emerald-400 bg-gray-200 px-4 py-2 text-sm text-sky-950 outline-none focus:ring-2 focus:ring-emerald-400 disabled:cursor-not-allowed disabled:bg-gray-400 disabled:text-gray-600;
+    padding-right: 2.5rem;
```

Stylelint (17.6.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)